Driver: Sensor: sht3xd_trigger thresholds corrected

louiscr opened the pull request:
SHT3XD_CMD_WRITE_TH_LOW_SET was set twice, should be SHT3XD_CMD_WRITE_TH_LOW_CLEAR once
Due to this set a wrong Clear threshold was set

Signed-off-by: Louis Creus <louis.creus@unitrongroup.com>

Fixes #48299

MaureenHelm reviewed (commented):
Thank you for the fix! The code change looks good, but you need to fix your commit message to pass the compliance check.

MaureenHelm commented:
Please rebase and remove the merge commits. You need to amend your original commit to follow the commit guidelines and force push your branch. See here for more details:
https://docs.zephyrproject.org/latest/contribute/guidelines.html

MaureenHelm commented:
I fixed your branch so it's now only one commit ahead of main, instead of the 500+ commits you had when something went wrong in a rebase. You still need to fix the commit message and force push (I can't add the signed-off-by for you).

Thanks again for the fix.

louisunitron committed:
Driver: Sensor: sht3xd_trigger thresholds corrected

Fix SHT3XD_CMD_WRITE_TH_LOW_SET set twice, should be SHT3XD_CMD_WRITE_TH_LOW_CLEAR
This set a wrong Clear threshold

Fixes #48299

Signed-off-by: Louis Creus louis.creus@telenet.be

cfriedt reviewed (commented):
The changes here are not really reflected in the summary. @louiscr - can you please rebase, squash, and make a more concise git commit?

cfriedt reviewed (changes requested):
see previous comment

MaureenHelm commented:
> I fixed your branch so it's now only one commit ahead of main, instead of the 500+ commits you had when something went wrong in a rebase. You still need to fix the commit message and force push (I can't add the signed-off-by for you).

Did this one more time. Please don't use the GitHub "Update branch" button because this adds merge commits to your branch. Instead you should rebase and amend commits locally at the command line, then force push.

Thanks for adding the signed-off-by line. Did you type it manually instead of using the git commit `-s` argument? Please see [DCO Sign-Off](https://docs.zephyrproject.org/latest/contribute/guidelines.html#dco-sign-off) for more details. There is still a problem though, because the author email doesn't match the signed-off-by email:
```
Author: Louis Creus <louis.creus@unitrongroup.com>
Date:   Thu Aug 18 14:29:02 2022 +0200

    Driver: Sensor: sht3xd_trigger thresholds corrected
    
    Fix SHT3XD_CMD_WRITE_TH_LOW_SET set twice, should be SHT3XD_CMD_WRITE_TH_LOW_CLEAR
    This set a wrong Clear threshold
    
    Fixes #48299
    
    Signed-off-by: Louis Creus louis.creus@telenet.be
```
